GitUI/CommandsDialogs/FormStash.Designer.cs

```diff
@@ -11,20 +11,6 @@ partial class FormStash
         /// </summary>
         private System.ComponentModel.IContainer components = null;
 
-        /// <summary>
-        /// Clean up any resources being used.
-        /// </summary>
-        /// <param name="disposing">true if managed resources should be disposed; otherwise, false.</param>
-        protected override void Dispose(bool disposing)
```

Comment:
💭 How did you keep it from generating this?

Reply:
I did it manually the same way as in a recently merged PR of an other contributor.